lib/util/git.js

```diff
@@ -25,6 +26,23 @@ const GOOD_ENV_VARS = new Set([
   'GIT_SSL_NO_VERIFY'
 ])
 
+const GIT_TRANSIENT_ERRORS = [
+  'remote error: Internal Server Error',
+  'fatal: Couldn\'t find remote ref ',
+  'The remote end hung up unexpectedly',
+  'Connection timed out',
+  'Operation timed out',
+  'Failed to connect to .* Timed out',
+  'Connection reset by peer',
+  'Couldn\'t resolve host'
+].join('|')
+
+const GIT_TRANSIENT_ERROR_RE = new RegExp(GIT_TRANSIENT_ERRORS)
+
+function shouldRetry(error) {
+  return GIT_TRANSIENT_ERROR_RE.test(error);
```

Comment:
This `;` made me realize the `standard` check was misconfigured -- it's now been fixed, and you'll need to fix that `standard` linter violations in this patch, too. Mostly, it's just removing the semicolons, I think. You'll get the errors now when you rebase onto `latest`.